tools/CouchDB_Import_MRI.php

```diff
@@ -111,6 +111,36 @@ function updateDataDict($types)
                 $this->feedback_PreDefinedComments[$CommentTypeID] = $pre;
 
             }
+            $this->mri_header_fields = array('ScannerID'=>'Scanner ID',
+                                       'Pipeline'=>'Pipeline',
+                                       'OutputType'=>'Output',
+                                       'AcquisitionProtocol'=>'Protocol',
+                                       'CoordinateSpace'=>'Space',
+                                       'Algorithm'=>'Algorithm',
+                                       'AcquisitionDate'=>'Acquisition Date',
+                                       'FileInsertDate'=>'Insert date',
+                                       'SeriesDescription'=>'Series Description',
+                                       'SeriesNumber'=>'Series Number',
+                                       'EchoTime'=>'Echo Time',
+                                       'RepetitionTime'=>'Repetition Time',
+                                       'SliceThickness'=>'Slice Thickness',
+                                       'Time'=>'No of volumes',
+                                       'Comment'=>'Comment',
+                                       'SlicewiseRejected'=>
+                                           'Slicewise correlations (Nb)',
+                                       'InterlaceRejected'=>
+                                           'Interlace correlations (Nb)',
+                                       'IntergradientRejected'=>
+                                           'Gradient-wise correlations (Nb)',
+                                       'TotalRejected'=>
+                                           'No. of rejected directions');
```

Comment:
Then it's either not pushed to GitHub, or not using the LorisCS.xml standard.. because the alignment of => and brackets is enforced by the LorisCS.xml file.